client/controllers/incidentReports/suggestedIncidentsModal.coffee

```diff
@@ -147,6 +138,18 @@ Template.suggestedIncidentsModal.onCreated ->
             Object.keys(timeAnnotation.timeRange.end).length +
             Object.keys(timeAnnotation.timeRange.end).length
           )
+          timeAnnotation.beginMoment = moment.utc(timeAnnotation.timeRange.begin)
+          # Round up the to day end
+          timeAnnotation.endMoment = moment.utc(_.extend({
```

Comment:
I think we can use the builtin .endOf('day') in momentJS here instead of rounding up like this

http://momentjs.com/docs/#/manipulating/end-of/